pyzabbix/api.py

```diff
@@ -40,6 +43,24 @@ def __init__(self, *args, **kwargs):
         self.error = kwargs.get("error", None)
 
 
+class ZabbixAPIAuth(AuthBase):
+    def __init__(self, token):
+        self.token = token
+
+    def __call__(self, request):
+        # At this point the JSON payload has already been serialized,
+        # we have to parse it again. Bummer.
+        js = json.loads(request.body)
+        if (self.token
+            and (not js
+                 or not "method" in js
+                 or js["method"] not in ("apiinfo.version",
+                                         "user.checkAuthentication",
+                                         "user.login"))):
+            request.headers["Authorization"] = "Bearer %s" % (self.token,)
+        return request
```

Comment:
Please move this out of the `AuthBase` class and put it next to the other version check condition.

I'd prefer having related things next to each other, and a auth class seem overkill for what we want to do.